Added ornament palette icon

sdziuda opened the pull request:
I did an icon for ornament palette and added it to artwork.js.

pikurasa commented:
I look forward to seeing the image.

Two things:

1)

I cannot get past the splashscreed. I get this error:

```
Uncaught ReferenceError: PALETTEFILLCOLORS is not defined
    at domReady (activity.js:342)
    at activity.js:182
    at Object.execCb (require.js:1604)
    at Module.check (require.js:844)
    at Module.<anonymous> (require.js:1084)
    at require.js:129
    at require.js:1127
    at each (require.js:57)
    at Module.emit (require.js:1126)
    at Module.check (require.js:898)
```

2) Please use Inkscapes "save as plain SVG" option to save the chosen SVG.

sdziuda commented:
Thank you. I've added the quotes and saved the icon as plain SVG. I also did how you told me and added a neighbor. Unfortunetly, I have no ideas about the error, because I haven't even changed anything in the activity.js or require.js.

walterbender commented:
The broken SVG would have broken artwork.js which would have meant certain defines were not made, hence the errors Devin observed.

walterbender commented:
Can you please change the color of the icon to the same color scheme as the tone icon?

sdziuda commented:
I've changed the color and added the 'ornament' line (140 and 165) in PALETTECOLORS0 and PALETTECOLORS and gave the same values as 'tone' has. I did it because I think that the error might occur because it was missing.
